BICI_Psoc_workspace/Fingertip_V3_acc.cydsn/main.c

```diff
@@ -11,6 +11,78 @@
 */
 #include <main.h>
 
+
+float X_out, Y_out, Z_out;

```

Comment:
Can you add this to the main.h if it's going to be global

Reply:
done